content/show/coder-radio/0476.md

```diff
@@ -55,11 +55,11 @@
   "podcast_chapters": null,
   "podcast_alt_file": null,
   "podcast_ogg_file": null,
-  "video_file": null,
+  "video_file": "https://201406.jb-dl.cdn.scaleengine.net/coderradio/2022/cr-0476.mp4",
```

Comment:
This link is not accessible for me, so something must have changed in the last two weeks? Or maybe it's just temporary?

Reply:
@ChanceM would you be able to update these Scale Engine links the HTTP? I think that makes the most sense for time being.